truss/templates/base.Dockerfile.jinja

```diff
@@ -4,12 +4,12 @@ FROM {{base_image_name_and_tag}}
 {% block fail_fast %}
 RUN grep -w 'ID=debian\|ID_LIKE=debian' /etc/os-release || { echo "ERROR: Supplied base image is not a debian image"; exit 1; }
 RUN pythonVersion=$(echo $(command -v python >/dev/null 2>&1 && python --version || python3 --version) | cut -d" " -f2 | cut -d"." -f1,2) \
-    && echo $pythonVersion | grep -E '3\.[0-9]|10\.[0-9][0-9]' \
+    && echo $pythonVersion | grep -E '3\.[0-9]|10|11\.[0-9][0-9]' \
```

Comment:
```suggestion
    && echo $pythonVersion | grep -E '3\.10|11|[0-9]\.[0-9][0-9]' \
```
to match `10` and `11` first instead of matching the first digit `1` with the pattern `[0-9]`